edx_proctoring/tests/test_api.py

```diff
@@ -1539,3 +1540,47 @@ def test_update_exam_attempt(self):
 
         self.assertEquals(attempt['last_poll_timestamp'], now)
         self.assertEquals(attempt['last_poll_ipaddr'], '1.1.1.1')
+
+    @ddt.data(
+        ProctoredExamStudentAttemptStatus.submitted,
+        ProctoredExamStudentAttemptStatus.verified,
+        ProctoredExamStudentAttemptStatus.rejected
+    )
+    def test_send_email(self, status):
+        """
+        Assert that email is sent on the following statuses of proctoring attempt.
+        """
+
+        exam_attempt = self._create_started_exam_attempt()
+        update_attempt_status(
+            exam_attempt.proctored_exam_id,
+            self.user.id,
+            status
+        )
+        self.assertEquals(len(mail.outbox), 1)
+        self.assertIn('Proctoring Session Results Update for', mail.outbox[0].subject)
+
```

Comment:
@chrisndodge 
I have added the subject check, we can add this check on the email body later. Because for now, I have not the exact template for email body.